apps/overlay-sandbox/app/page.tsx

```diff
@@ -3,7 +3,7 @@ import Link from 'next/link';
 export default async function Index() {
   return (
     <div>
-      <Link href={'/cases/overlay'}>Overlay</Link>
+      <Link href={'/cases/overlay/modelIdSetSandbox'}>Overlay</Link>
```

Comment:
Why do you move it in `/modelIdSetSandbox`? At least it should be in kebab-case as `overlay-manager`, but I think previous `/cases/overlay` was better

Reply:
for the testing purposes sandboxes with different options and combination of feature flags should be launched. I gonna put all the sandboxes inside `/cases/overlay/` that's why `modelIdSetSandbox`, which is used for verification of `modelId` option, was placed there. 
Fixed incorrect case